keyboards/leafcutterlabs/bigknobv2/readme.md

```diff
@@ -0,0 +1 @@
+# bigKNOBv2
```

Comment:
Could you create a nice readme with picture, compilation and reset information?

Check out the other boards' readmes. (e.g.: https://github.com/qmk/qmk_firmware/blob/master/keyboards/planck/readme.md)